tasks/weblogic_startup_service.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Create service startup template for admin server
+  become: yes
+  become_user: "root"
+  template:
+    src: 'weblogic.service.j2'
+    dest: '/etc/systemd/system/weblogic.service'
+    mode: '0644'
+    owner: "root"
+    group: "root"
+
+- name: Reload the (new) service which will also start it
+  become: yes
+  become_user: "root"
+  systemd:
+    name: weblogic.service
+    state: reloaded
```

Comment:
`state: stopped` rather, we don't want this running yet in the ami during the build, we need to configure it and start it during the bootstrap